src/main/kotlin/no/fdk/fdk_informationmodel_harvester/rdf/RDFUtils.kt

```diff
@@ -56,9 +47,9 @@ fun Statement.isResourceProperty(): Boolean =
         false
     }
 
-fun Model.createRDFResponse(responseType: JenaType): String =
+fun Model.createRDFResponse(responseType: Lang): String =
     ByteArrayOutputStream().use{ out ->
-        write(out, responseType.value)
+        write(out, responseType.name)
```

Comment:
Ganske sikker på at det skal også funke:
```suggestion
        write(out, responseType)
```

Reply:
Samme problem som ved read. `Model.write` tar kun inn lang som String